internal/git/git.go

```diff
@@ -115,6 +115,27 @@ func (r *Repo) Run(opts *RunOpts) (*Output, error) {
 	}, nil
 }
 
+// RunInteractive is similar to Run, but directly connects the standard I/Os to the console,
+// allowing the git command to interact with the user.
+func (r *Repo) RunInteractive(opts *RunOpts) (int, error) {
```

Comment:
Can you just make this an option on `RunOpts` rather than introducing a new, mostly duplicated function? The returned stdout/stderr would always be an empty byte array but that's fine.

Reply:
Done